#!/bin/sh /usr/share/dpatch/dpatch-run
## 03-config-kmk.dpatch by Patrick Winnertz <patrick.winnertz@skolelinux.org>
##
## DP: Some patches for Config.kmk to build with the debian tools

@DPATCH@

diff -Naur virtualbox-ose-1.5.0-dfsg.orig/Config.kmk virtualbox-ose-1.5.0-dfsg/Config.kmk
--- virtualbox-ose-1.5.0-dfsg.orig/Config.kmk	2007-08-31 13:01:30.000000000 +0000
+++ virtualbox-ose-1.5.0-dfsg/Config.kmk	2007-09-05 05:10:53.000000000 +0000
@@ -838,7 +838,7 @@
 # We need to export YASM for OSE, but since it was converted to .zip we need to
 # allow for the location used by the zip to be properly cleaned up first.
 ifeq ($(filter-out win.x86 linux.x86 solaris.x86, $(BUILD_PLATFORM).$(BUILD_PLATFORM_ARCH)),)
- PATH_TOOL_YASM ?= $(PATH_DEVTOOLS_BLD)/bin
+ PATH_TOOL_YASM ?= /usr/bin
 endif
 
 # XPCOM setup for all but win32.
@@ -1199,7 +1199,7 @@
   kernelsrc := $(shell $(SED_EXT) -ne 's+^KERNELSRC.*= *\(.*\)+\1+p' $(VBOX_LINUX_SRC)/Makefile)
   ifneq ($(kernelsrc),)
    kernelpath := $(realpath $(VBOX_LINUX_SRC)/$(kernelsrc))
-   VBOX_LINUX_INCS                 += $(VBOX_LINUX_SRC)/include2 $(kernelpath)/include
+   VBOX_LINUX_INCS                 += $(VBOX_LINUX_SRC)/include2 /usr/include
   else
    kernelpath := $(VBOX_LINUX_SRC)
   endif
